# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/tests/gtest/TestGfxPrefs.cpp
# Commit: 898ed02804fe
# Full Hash: 898ed02804fefc9786112f135b9621cc444e6860
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r e1b7abc99ae9 -r 898ed02804fe gfx/tests/gtest/TestGfxPrefs.cpp
--- a/gfx/tests/gtest/TestGfxPrefs.cpp	Wed May 22 12:44:04 2019 +0000
+++ b/gfx/tests/gtest/TestGfxPrefs.cpp	Wed May 22 22:27:37 2019 +0000
@@ -28,16 +28,16 @@
   ASSERT_TRUE(gfxPrefs::SingletonExists());
 
   // Once boolean, default true
-  ASSERT_TRUE(gfxPrefs::WorkAroundDriverBugs());
+  ASSERT_TRUE(StaticPrefs::WorkAroundDriverBugs());
 
   // Once boolean, default false
-  ASSERT_FALSE(gfxPrefs::UseApitrace());
+  ASSERT_FALSE(StaticPrefs::UseApitrace());
 
   // Once uint32_t, default 5
-  ASSERT_TRUE(gfxPrefs::APZMaxVelocityQueueSize() == 5);
+  ASSERT_TRUE(StaticPrefs::APZMaxVelocityQueueSize() == 5);
 
   // Once float, default 0 (should be OK with ==)
-  ASSERT_TRUE(gfxPrefs::APZCurveFunctionX1() == 0.0f);
+  ASSERT_TRUE(StaticPrefs::APZCurveFunctionX1() == 0.0f);
 }
 
 TEST(GfxPrefs, Set)
@@ -46,18 +46,18 @@
   ASSERT_TRUE(gfxPrefs::SingletonExists());
 
   // Once boolean, default false
-  ASSERT_FALSE(gfxPrefs::UseApitrace());
-  gfxPrefs::SetUseApitrace(true);
-  ASSERT_TRUE(gfxPrefs::UseApitrace());
-  gfxPrefs::SetUseApitrace(false);
-  ASSERT_FALSE(gfxPrefs::UseApitrace());
+  ASSERT_FALSE(StaticPrefs::UseApitrace());
+  StaticPrefs::SetUseApitrace(true);
+  ASSERT_TRUE(StaticPrefs::UseApitrace());
+  StaticPrefs::SetUseApitrace(false);
+  ASSERT_FALSE(StaticPrefs::UseApitrace());
 
   // Once float, default 0
-  ASSERT_TRUE(gfxPrefs::APZCurveFunctionX1() == 0.0f);
-  gfxPrefs::SetAPZCurveFunctionX1(1.75f);
-  ASSERT_TRUE(gfxPrefs::APZCurveFunctionX1() == 1.75f);
-  gfxPrefs::SetAPZCurveFunctionX1(0.0f);
-  ASSERT_TRUE(gfxPrefs::APZCurveFunctionX1() == 0.0f);
+  ASSERT_TRUE(StaticPrefs::APZCurveFunctionX1() == 0.0f);
+  StaticPrefs::SetAPZCurveFunctionX1(1.75f);
+  ASSERT_TRUE(StaticPrefs::APZCurveFunctionX1() == 1.75f);
+  StaticPrefs::SetAPZCurveFunctionX1(0.0f);
+  ASSERT_TRUE(StaticPrefs::APZCurveFunctionX1() == 0.0f);
 }
 
 // Randomly test the function we use in nsExceptionHandler.cpp here: